src/test/functional/pages/BuzzPage.ts

```diff
@@ -41,4 +48,12 @@ export class BuzzPage extends BasePage {
         await this.deletePostParagraph.click();
         await this.confirmDeleteButton.click();
     }
+
+    async editPost(postEdited:string): Promise<void> {
```

Comment:
To be honest - it was a bit confusing to me. editPost("post edited") is suggesting that you will edit post with name "post edited". To avoid confusion I'm suggesting:
If you need to remove a picture as a part of an edition - perhaps it should be parametrize to show person that reads a test that it will do that. I mean If i read name editPost("final text") I don't expect the method to provide any other changes...
`public async editPost(finalPostText: string, removePicture: boolean): Promise<void> {`